views/fxa_authorize.js

```diff
@@ -25,9 +25,10 @@ define('views/fxa_authorize', ['capabilities', 'log', 'login', 'utils', 'z'],
             var packaged_origin = 'app://packaged.' + window.location.host;
             console.log('Sending OAuth code to parent window ' +
                         packaged_origin);
+            // Don't remove the splash screen for this view;
+            z.context.dont_hide_splash = true;
+            // The opener will take responsibility of closing the window.
             window.opener.postMessage({auth_code: auth_code}, packaged_origin);
-
-            window.close();
```

Comment:
That's taken care of here: https://github.com/mozilla/marketplace-core-modules/pull/15/files#diff-9a78cc20f3732fc9666d3a2f5087793eR273